plugins/core/src/main/scala/org/broadinstitute/monster/sbt/MonsterBasePlugin.scala

```diff
@@ -26,30 +26,48 @@ object MonsterBasePlugin extends AutoPlugin {
   override def requires: Plugins =
     JvmPlugin && DynVerPlugin && ScalafmtPlugin && ScoverageSbtPlugin && BuildInfoPlugin
 
-  val ScalafmtVersion = "2.2.2"
+  val ScalafmtVersion = "2.4.2"
 
   val ScalafmtConf: String =
-    s"""version = "$ScalafmtVersion"
-       |maxColumn = 90
-       |runner.optimizer.forceConfigStyleOnOffset = 90
+    s"""version = $ScalafmtVersion
        |
+       |# Line-width settings.
+       |maxColumn = 100
+       |runner.optimizer.forceConfigStyleOnOffset = 100
```

Comment:
I noticed discussion about formatting in [this comment thread](https://github.com/DataBiosphere/encode-ingest/pull/23#discussion_r398103477). The rules used to auto-format are defined in this string. The specific issue talked about in encode-ingest was caused by the "max line length". I've been feeling like it was annoyingly strict for awhile now, so I bumped it up by 10 chars here.

Reply:
Yay! That's been annoying me a bit too... I am looking forward to those 10 extra characters.

Reply:
Ooh I didn't know those rules were set by the plugin. Thank you for bumping this up!